shared_model/interfaces/iroha_internal/transaction_batch_impl.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef IROHA_TRANSACTION_BATCH_IMPL_HPP
+#define IROHA_TRANSACTION_BATCH_IMPL_HPP
+
+#include "interfaces/iroha_internal/transaction_batch.hpp"
+
+namespace shared_model {
+  namespace interface {
+
+    class TransactionBatchImpl : public TransactionBatch {
+     public:
+      TransactionBatchImpl() = delete;
```

Comment:
Is it necessary to delete the default constructor if another ctor is already declared? https://github.com/hyperledger/iroha/blob/e7255a34b174b4efb26151a1c7e14cb0eab36125/shared_model/interfaces/iroha_internal/transaction_batch_impl.hpp#L23-L24